src/utils/themes-manager.js

```diff
@@ -0,0 +1,79 @@
+//@flow
+
+import {hexToHsl} from './color-format-convertors';
+import {style} from '../index';
+
+const cssVarNames: ThemesManagerConfig = {
+  colors: {
+    primary: '--playkit-primary-hsl-hue',
+    secondary: '--playkit-secondary-hsl-hue',
+    success: '--playkit-success-hsl-hue',
+    danger: '--playkit-danger-hsl-hue',
+    warning: '--playkit-warning-hsl-hue',
+    live: '--playkit-live-color',
+    playerBackground: '--playkit-player-background-color'
+  }
+};
+
+const dynamicColoredIconsSvgUrlVars = [
+  '--playkit-icon-check-active-url',
+  '--playkit-icon-data-url',
+  '--playkit-icon-chromecast-url',
+  '--playkit-icon-quality-HD-active-url',
+  '--playkit-icon-quality-4K-active-url',
+  '--playkit-icon-quality-8K-active-url'
+];
+
+// eslint-disable-next-line require-jsdoc
+export class ThemesManager {
+  player: Object;
+  config: Object;
+
+  // eslint-disable-next-line require-jsdoc
+  constructor(player: Object, config: UserTheme) {
+    this.player = player;
+    this.config = config;
+  }
+
+  // eslint-disable-next-line require-jsdoc
+  applyUserTheme(): void {
+    if (this.config) {
```

Comment:
can this be not set ?

Reply:
yes it does, the default colors comes from the css files and bundled with the player , the config just override